src/utils/treeUtils.ts

```diff
@@ -22,4 +22,8 @@ export namespace treeUtils {
     function getResourcesPath(): string {
         return ext.context.asAbsolutePath('resources');
     }
+
+    export function isAzExtTreeItem(ti: unknown): ti is AzExtTreeItem {
```

Comment:
We're planning to add this function to the core utils library (https://github.com/microsoft/vscode-azuretools/pull/1229/files#diff-a94c5ee42a49df43d3b3624aea93155299963c5571dcefed509b755d942640ae). Can we either move that function now, so this PR can use it, or otherwise make a note in the code here to start using that function from utils when it is available?

Reply:
Exists here too https://github.com/microsoft/vscode-azuretools/blob/main/appservice/src/deploy/getDeployNode.ts#L13